### ISSUE TITLE
I need docker-machine to use my ssh config file

### ISSUE THREAD

**Issue by lookitup4me** [docker/machine#3794]
Currently https://github.com/docker/machine/blob/df2d3811ca8bc9ddf6896b4a4154b9277826b441/libmachine/ssh/client.go#L69 forces docker-machine to ignore the ssh config. It says 
```
	baseSSHArgs = []string{
		"-F", "/dev/null",
    ...

```
I need docker-machine to essentially read "~/.ssh/config", the standard config file. How can I override this default setting?

**Comment by nathanleclaire** [comment]
The source code would need to be modified to support it... I think there might be a PR/proposal in flight for this already

**Comment by lookitup4me** [comment]
@nathanleclaire you are right. There is a PR indeed. https://github.com/docker/machine/pull/3410

**Comment by mattjudge** [comment]
Any update on this? It seems a bizarre default setup, to explicitly force ssh to ignore the user's settings. I'm trying to use docker-machine on a box behind a corporate proxy, and this is preventing me from doing so.
